### PR TITLE
xdman: Update to version 7.2.11

### PR DESCRIPTION
**Pull request by omerakgoz34**
- Closes #3000

### CODE REVIEW

File: bucket/xdman.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
     "description": "A powerful FOSS downloader supports protocols as HTTP,HTTPS,FTP,MPEG-DASH,HLS,HDS and features as seamlessly integration with browsers, monitoring and capture, broken/dead tasks resumption",
-    "homepage": "http://xdman.sourceforge.net/",
-    "license": "Unknown",
-    "url": "https://downloads.sourceforge.net/project/xdman/xdmsetup-2018.msi",
-    "hash": "sha1:506c8b1014aa1ea17979c548aa7c4dd1bd110f14",
+    "homepage": "https://subhra74.github.io",
+    "license": "GPL-2.0-only",
+    "url": "https://github.com/subhra74/xdm/releases/download/7.2.11/xdm-setup.msi",
+    "hash": "dce85ee5d7d38894c7dc2b4042d94dc087814dbbdb00d0995149a5ecc481d25c",
     "extract_dir": "XDM",
```
Comment (Ash258) [inline bucket/xdman.json:8]:
```suggestion
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/xdman.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
             "Xtreme Download Manager"
         ]
     ],
-    "checkver": "XDM\\s+(?<year>\\d{4})\\s+Version\\s+(?<version>[\\d.]+)",
+    "checkver": "github",
     "autoupdate": {
-        "url": "https://downloads.sourceforge.net/project/xdman/xdmsetup-$matchYear.msi"
+        "url": "https://github.com/subhra74/xdm/releases/download/$version/xdm-setup.msi"
```
[the remark this answers is not in the document]
Comment (omerakgoz34) [reply bucket/xdman.json:27]: idk

**Comment by omerakgoz34** [comment]
@Ash258 
These functions do not work without exe in the same folder. (video download, FFmpeg, etc...)
Screenshot_2

**Comment by Ash258** [comment]
I wish there would be a way how to install ffmpeg and youtube-dl with one simple commmand.

There is no reason to download full msi to have just ffmpeg, youtube-dl and non working java jre.

Find out why there are two msi setups and then I can consider using one them.

File: bucket/xdman.json [hunk 3]
```diff
@@ -1,21 +1,42 @@
...
+    "url": "https://github.com/subhra74/xdm/releases/download/7.2.11/xdman.jar",
+    "hash": "8117f286f9de1e57f4551676d8f0ab3e2e990444744b595f7cc4cb9e01f51c7c",
+    "pre_install": [
+        "Set-Content \"$dir\\xdman.bat\" '@start javaw.exe -jar \"%~dp0xdman.jar\" %*' -Encoding Ascii"
+    ],
+    "post_install": [
+        "cmd /c '@echo off && mklink \"%AppData%\\..\\..\\scoop\\apps\\xdman\\current\\ffmpeg.exe\" \"%AppData%\\..\\..\\scoop\\shims\\ffmpeg.exe\" >nul 2>&1'",
+        "cmd /c '@echo off && mklink \"%AppData%\\..\\..\\scoop\\apps\\xdman\\current\\ffmpeg.shim\" \"%AppData%\\..\\..\\scoop\\shims\\ffmpeg.shim\" >nul 2>&1'",
+        "cmd /c '@echo off && mklink \"%AppData%\\..\\..\\scoop\\apps\\xdman\\current\\youtube-dl.exe\" \"%AppData%\\..\\..\\scoop\\shims\\youtube-dl.exe\" >nul 2>&1'",
+        "cmd /c '@echo off && mklink \"%AppData%\\..\\..\\scoop\\apps\\xdman\\current\\youtube-dl.shim\" \"%AppData%\\..\\..\\scoop\\shims\\youtube-dl.shim\" >nul 2>&1'"
```
Comment (Ash258) [inline bucket/xdman.json:27]: - Completely nonsense assumption that everyone has scoop userprofile and ignoring global installations
- Absolutely ugly solution to "problem" which noone came in

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/xdman.json (hunk 3 above)
Comment (omerakgoz34) [reply bucket/xdman.json:27]: Unfortunately, xdman doesn't use exe files from global PATH :/
I couldn't find any way to solve this except using an absolute path to exe files
What is your suggestion to solve this "problem" ?

Comment (omerakgoz34) [reply bucket/xdman.json:27]: > * Completely nonsense assumption that everyone has scoop userprofile and ignoring global installations
> * Absolutely ugly solution to "problem" which noone came in

Unfortunately, xdman doesn't use exe files from global PATH :/
I couldn't find any way to solve this except using an absolute path to exe files
What is your suggestion to solve this "problem" ?

Comment (omerakgoz34) [reply bucket/xdman.json:27]: btw, the only difference between the two setup files is the FFmpeg version (xdmsetup.msi uses master branch version)

**Comment by omerakgoz34** [comment]
.d

**Comment by candrapersada** [comment]
why doesn't xdman use javaw.exe in the Portable folder?
`jre\bin\javaw.exe`
image
image
image

**Pull request closed without merging**